_pages/index.md

```diff
@@ -1,70 +1,33 @@
 ---
 layout: page
 permalink: /
-title: About Open Government in DC
+title: Open Government in DC
 ---
 
-The District is a strong advocate for open government and provides a large quantity of government information available to the public. If the information you are looking for is not already available on DC.Gov, you may [submit a FOIA request](http://dc.gov/node/720382).
+The District is a strong advocate for open government and provides a large quantity of government information available to the public. 
 
-## Draft Open Data Policy
+On this site you can learn more about DC's Open Government policies and open source projects, how to find government data and information, and who to contact for more.
 
-The District has released a [draft open data policy](https://drafts.dc.gov/docs/draft-open-data-policy) that is open for comment until February 15, 2015. Find the policy and comment on it at [drafts.dc.gov](https://drafts.dc.gov/docs/draft-open-data-policy). Comments are also accepted by:
+***
 
-* Sending an email to [open@dc.gov](mailto:open@dc.gov)
-* Tweeting questions and ideas to [@octodc](https://twitter.com/OCTODC) on Twitter using the hashtag #opendatadc
-* Mailing comments c/o Open Data Policy, 200 I Street SE, Suite 500, Washington, DC 20003
+# [Policy >>](./policy)
 
-## Government Performance
+Building off of the 2014 Open Government and Open Data Directive, DC Government is working on a new [Open Data Policy]().
```

Comment:
is the link missing, or is this an implicit anchor link? I don't know this markdown that well